Sources/Build/llbuild.swift

```diff
@@ -304,20 +303,28 @@ public struct LLBuildManifestGenerator {
         })
 
         // For Clang, the target requires all command outputs.
-        var buildTarget = Target(name: target.target.llbuildTargetName)            
+        var buildTarget = Target(name: target.target.getBuildTargetName(config: plan.buildParameters.configuration.dirname))
         buildTarget.outputs.insert(contentsOf: commands.flatMap({ $0.tool.outputs }))
         buildTarget.cmds += commands
         return buildTarget
     }
 }
 
 extension ResolvedTarget {
-    public var llbuildTargetName: String {
-        return "\(name).module"
+    public func getCommandName(config: String?) -> String {
+       return "C." + getBuildTargetName(config: config)
     }
-
-    var commandName: String {
-        return "C.\(llbuildTargetName)"
+    
+    public func getBuildTargetName(config: String?) -> String {
```

Comment:
This shouldn't be optional. We should always pass the config.

Reply:
nit: function name -> `getLLBuildTargetName(config:)`

Reply:
Done